packages/client-engine-runtime/src/transactionManager/TransactionManager.ts

```diff
@@ -62,7 +61,7 @@ export class TransactionManager {
     const validatedOptions = this.validateOptions(options)
 
     const transaction: TransactionWrapper = {
-      id: crypto.randomUUID(),
+      id: webcrypto.randomUUID(),
```

Comment:
The following is available in the minimum Node.js version we require, 18.18.x:
```suggestion
      id: globalThis.crypto.randomUUID(),
```